.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
+  - python setup.py build
+  - python setup.py install
```

Comment:
`setup.py install` is deprecated (uses setuptools's `easy_install` command, produces an `.egg` file), you should use pip to install, which will internally create a wheel and install from there.

You can install from a local directory (e.g. the current one where setup.py is located) with `pip install .`